lib/config/migrations/custom/package-rules-migration.ts

```diff
@@ -0,0 +1,36 @@
+import type { PackageRule } from '../../types';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export const renameMap = {
+  paths: 'matchPaths',
+  languages: 'matchLanguages',
+  baseBranchList: 'matchBaseBranches',
+  managers: 'matchManagers',
+  datasources: 'matchDatasources',
+  depTypeList: 'matchDepTypes',
+  packageNames: 'matchPackageNames',
+  packagePatterns: 'matchPackagePatterns',
+  sourceUrlPrefixes: 'matchSourceUrlPrefixes',
+  updateTypes: 'matchUpdateTypes',
+};
+
+export class PackageRulesMigration extends AbstractMigration {
+  override readonly propertyName = 'packageRules';
+
+  override run(value: PackageRule[] | null): void {
```

Comment:
`value` is unused here, and shares a name with the key-value pair destructuring below. Is it required?

Reply:
Not necessary maybe left it there by mistake.